service-deployment/clusterconfig-example.yaml

```diff
@@ -85,6 +85,18 @@ clusterinfo:
     cluster_status_uri: http://10.0.3.9:9090
     # port for webportal
     server_port: 9286
+  
+  prometheusinfo:
+    # The address of the prometheus to connect to
+    prometheus_url: http://10.0.3.9:9090
+    # port for prometheus port
+    prometheus_port: 9090
+  
+  grafanainfo:  
+    # The address of the grafana to connect to
+    grafana_url: http://10.0.3.9:3000
```

Comment:
Why do you set the port info twice here? Duplicated configuration?

Reply:
Solved